tests/cnf/core/network/metallb/tests/metallb-crds.go

```diff
@@ -66,18 +69,18 @@ var _ = Describe("MetalLb New CRDs", Ordered, Label("newcrds"), ContinueOnFailur
 
 		By("Retrieving external IPv4 addresses of worker nodes")
 		ipv4NodeAddrList, err := nodes.ListExternalIPv4Networks(
-			APIClient, metav1.ListOptions{LabelSelector: labels.Set(NetConfig.WorkerLabelMap).String()})
+			APIClient, metav1.ListOptions{LabelSelector: labels.Set(workerLabelMap).String()})
 		Expect(err).ToNot(HaveOccurred(), "Failed to collect external nodes ip addresses")
 
 		By("Setup MetalLB CR")
-		err = metallbenv.CreateNewMetalLbDaemonSetAndWaitUntilItsRunning(tsparams.DefaultTimeout, NetConfig.WorkerLabelMap)
+		err = metallbenv.CreateNewMetalLbDaemonSetAndWaitUntilItsRunning(tsparams.DefaultTimeout, workerLabelMap)
 		Expect(err).ToNot(HaveOccurred(), "Failed create MetalLB CR")
 
 		By("Creating nginx test pod")
-		setupNGNXPod(cnfWorkerNodeList[0].Definition.Name, tsparams.LabelValue1)
+		setupNGNXPod(workerNodeList[0].Definition.Name, tsparams.LabelValue1)
 
 		By("Generating ConfigMap configuration for the external FRR pod")
-		masterConfigMap := createConfigMap(tsparams.LocalBGPASN, ipv4NodeAddrList, false, false)
+		masterConfigMap := createConfigMap(tsparams.RemoteBGPASN, ipv4NodeAddrList, false, true)
```

Comment:
Was the Local AS configured incorrectly?

Reply:
It was a workaround for the BGP+BFD bug, that was fixed recently in 4.19. Reverted it back